### ISSUE TITLE
Error: Field must be a float array

### ISSUE THREAD

**Issue by fabien** [typesense/typesense#115]
## Description

When inserting documents with an array of numbers (through the JS client), numbers that are actually integers cause an error to be thrown for `float[]` fields.

## Steps to reproduce

```javascript

let document = {
  'id': '124',
  'values': [ 20 ]
}

client.collections('companies').documents().create(document);
```
## Expected Behavior

Any numeric values that happen to be integers should be implicitly treated as floats for these fields. Javascript doesn't have proper floats, and numbers cannot be typed as floats specifically.

## Actual Behavior

Error when indexing such a document:

```
Field `values` must be a float array.
```

## Metadata

**Typsense Version**: 0.13.0

**OS**: Docker

**Comment by kishorenc** [comment]
Thank you for reporting this issue. Fixed in https://github.com/typesense/typesense/commit/ecf8e0c9f94a50939cf43fb7c6c7d18fbb9b6601 and the fix is available in this Docker image: `typesense/typesense:e16a428`.

**Comment by kishorenc** [comment]
Fixed in `v0.1.4.0` just releases.
